spec/takeaway_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../lib/takeaway'
+
+describe Takeaway do
+
+  describe '#menu returns itself' do
+    it 'prints menu when called' do
+      expect(subject.display_menu).to eq 'Fish' => 1, 
```

Comment:
use of subject throughout the spec. It's not wrong, but you can define a name to subject e.g subject(:takeaway) { described_class.new}